config/runtime/tests/preempt-rt.jinja2

```diff
@@ -0,0 +1,44 @@
+- test:
+    name: {{ node.name }}
+    timeout:
+      minutes: {{ job_timeout }}
+
+    definitions:
+    - from: inline
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: preempt-rt-prereq
+          description: Pre-requisites for PREEMPT_RT
+        run:
+          steps:
+          - apt-get update && apt-get install -y procps rt-tests
+      name: preempt-rt-prereq
+      path: inline/{{ node.name }}.yaml
```

Comment:
nit: you could safely leave the `path` from imported definition fragment ("inline/preempt-rt-prereq.yaml") - for `inline` definitions it's there only to comply with YAML structure. Let's leave this as it is now

Reply:
Sure.